src/cloud-cli/applications/deploy-app-code.ts

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+import YAML from "yaml";
+import { execSync } from "child_process";
+import fs from "fs";
+import FormData from "form-data";
+import { createGlobalLogger } from "../../telemetry/logs";
+import { getCloudCredentials } from "../utils";
+import { ConfigFile, parseConfigFile, operonConfigFilePath } from "../../operon-runtime/config";
+
+export async function deployAppCode(appName: string, host: string, port: string) {
+  const logger = createGlobalLogger();
+  const userCredentials = getCloudCredentials();
+  const bearerToken = "Bearer " + userCredentials.token;
+
+  try {
+    execSync(`mkdir -p operon_deploy`);
```

Comment:
Shelling out and not paying attention to any error messages - boo!

import { promises as fs } from 'fs';

async function createDirectory(path: string): Promise<void> {
    await fs.mkdir(path, { recursive: true });
}

Reply:
good idea, thanks!